sites/fast-component-explorer/app/web-components/index.tsx

```diff
@@ -0,0 +1,102 @@
+/** @jsx h */ /* Note: Set the JSX pragma to the wrapped version of createElement */
+
+import React from "react";
+import {
+    FASTDesignSystemProvider,
+    FASTTab,
+    FASTTabPanel,
+    FASTTabs,
+} from "@microsoft/fast-components";
+import { neutralLayerL1, neutralLayerL3 } from "@microsoft/fast-components-styles-msft";
+import h from "./pragma";
+
+/**
+ * Ensure tree-shaking doesn't remove these components from the bundle
+ */
+FASTDesignSystemProvider;
+FASTTab;
+FASTTabPanel;
+FASTTabs;
+
+interface RenderDevToolsTabsConfig {
+    codeRenderCallback: (e: HTMLElement) => void;
+    tabUpdateCallback: (e: React.ChangeEvent<HTMLElement>) => void;
+    guidanceTabPanelContent: React.ComponentClass<{}, any>;
+    definitionTabPanelContent: string;
+    schemaTabPanelContent: string;
+}
+
+export function renderDevToolsTabs(config: RenderDevToolsTabsConfig): React.ReactNode {
+    const styleOverrides: string = `
+        a {
+            color: var(--accent-foreground-rest);
+        }
```

Comment:
Does this already have hover/focus behaviors applied? If not - does it need them?

Reply:
We don't have a hover on it currently, could add it in future when we take another design pass.